src/components/Areas/Areas.scss

```diff
@@ -1,12 +1,17 @@
 .area-card {
-  border: 5px solid black;
-  height: 300px;
+  border: .5em solid black;
```

Comment:
Good job using `em` as a measurement here! Maybe not for this project, but take a look into what the differences are between `em` and `rem`. Are there benefits/pitfalls to one over the other?

Reply:
Definitely! I tend to stick to `ems` so things scale proportionally to what's around them, rather than just proportional to page via `rem`. For a border like this though I could see benefit to using `rem` so the borders are always relational to page font size.